Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -781,7 +781,7 @@ static void registerWithAccessibility()
 
 #endif // USE(OS_STATE)
 
-#if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
+#if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT) && PLATFORM(IOS_FAMILY)
```

Comment:
It looks like this condition is used for multiple functions including `registerLogHook()`. Should we only change it for `prewarmLogs()`?

Reply:
Ah, good catch, fixed!

Thanks for reviewing!